e2etest/get_place_statsvar_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2etest
+
+import (
+	"context"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+	"github.com/datacommonsorg/mixer/server"
+)
+
+func TestGetPlaceStatsVar(t *testing.T) {
+	ctx := context.Background()
+	client, err := setup(server.NewMemcache(map[string][]byte{}))
+	if err != nil {
+		t.Fatalf("Failed to set up mixer and client")
+	}
+
+	dcids := []string{"geoId/05", "geoId/06885", "invalid"}
+	want := map[string]struct {
+		minCount  int
```

Comment:
Why are we requiring minCount (as opposed to max like in our other API calls)

Reply:
This is to check the returned result has a minimal amount stats vars, mostly for testing the api is working.

Reply:
Ah, ok. It's just for the test..